tests/test.sh

```diff
@@ -97,6 +97,15 @@ test_jsn() {
   $CURL "$URL" | jq --sort-keys -e 'walk(if type == "number" then .+0.0 else . end)' > "$FILENAME"
 }
 
+test_metrics() {
+  FILENAME="$TEST_OUT_DIR/$1"
+  URL="$MARTIN_URL/metrics"
+
+  echo "Testing $(basename "$FILENAME") from $URL"
+  $CURL "$URL" | sed -E 's/^(martin_.*?) [\.0-9]+$/\1 NUMBER/g' > "$FILENAME"
+  $CURL --compressed "$URL" | sed -E 's/^(martin_.*?) [\.0-9]+$/\1 NUMBER/g' > "$FILENAME"
```

Comment:
I added a few testcases to verify this works and came upon this:
- if I request WITH compression, a the tile `metrics`
- if I request WITHOUT compression, I get `/metrics`

Will investigate further later in the week